app/src/main/java/com/github/mobile/ui/issue/IssuesViewActivity.java

```diff
@@ -330,10 +330,7 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 }
             }
             if (repository != null) {
-                Intent intent = RepositoryViewActivity.createIntent(repository);
-                intent.addFlags(FLAG_ACTIVITY_CLEAR_TOP
-                        | FLAG_ACTIVITY_SINGLE_TOP);
-                startActivity(intent);
+                finish();
```

Comment:
This change fix the bug of wrong action when back button pressed (https://github.com/AndreySBer/ForkHub/issues/71)
When we open this Activity and press back button (arrow to the left on screen) redirects always to repository activity with tabs instead of previous (in our case milestone) view.

Reply:
You're mixing back and up navigation (https://developer.android.com/design/patterns/navigation.html). That's not a back button.